app/src/main/java/com/github/se/eduverse/ui/pdfGenerator/PdfGenerator.kt

```diff
@@ -174,18 +180,42 @@ fun PdfGeneratorScreen(
                                 PdfGeneratorViewModel.PdfGenerationState.Ready)
                   }
 
-              OptionCard(
-                  testTag = PdfGeneratorOption.EXTRACT_TEXT.name,
-                  optionName = "Extract text",
-                  explanation = "Extracts text from an image",
-                  icon = Icons.Default.Abc,
-                  onClick = {
-                    currentPdfGeneratorOption = PdfGeneratorOption.EXTRACT_TEXT
-                    inputFileMIMEType = "image/*"
-                    showInfoWindow = true
-                  },
-                  optionEnabled =
-                      pdfConversionState.value == PdfGeneratorViewModel.PdfGenerationState.Ready)
+              Row(
+                  modifier = Modifier.fillMaxWidth(),
+                  horizontalArrangement = Arrangement.SpaceEvenly) {
+                    OptionCard(
+                        testTag = PdfGeneratorOption.EXTRACT_TEXT.name,
+                        optionName = "Extract text",
+                        explanation = "Extracts text from an image",
+                        icon = Icons.Default.Abc,
+                        onClick = {
+                          currentPdfGeneratorOption = PdfGeneratorOption.EXTRACT_TEXT
+                          inputFileMIMEType = "image/*"
+                          showInfoWindow = true
+                        },
+                        optionEnabled =
+                            pdfConversionState.value ==
+                                PdfGeneratorViewModel.PdfGenerationState.Ready)
+
+                    OptionCard(
+                        testTag = PdfGeneratorOption.TRANSCRIBE_SPEECH.name,
+                        optionName = "Speech to PDF",
+                        explanation = "Transcribes speech to text",
+                        icon = Icons.Default.Mic,
+                        onClick = {
+                          if (SpeechRecognizer.isRecognitionAvailable(context)) {
+                            currentPdfGeneratorOption = PdfGeneratorOption.TRANSCRIBE_SPEECH
+                            showTranscriptionDialog = true
+                          } else {
+                            // Show toast if speech recognition is not provided by the device
+                            context.showToast(
+                                "Cannot use this tool. Speech recognition not supported by your device.")
+                          }
+                        },
+                        optionEnabled =
+                            pdfConversionState.value ==
+                                PdfGeneratorViewModel.PdfGenerationState.Ready)
+                  }
```

Comment:
Make sure to follow the project design that have been updated, please see my pull request 299, indeed after having talked to some team members they agreed to keep the design done in 299. Here you put 2 OptionCard's in the same row, while only one should be per row, also you don't display the visual gradient animation. Please make sure to do changes accordingly to the actual design.

Reply:
Yes I didn't do the merge yet, I just created the PR to see the coverage. But it will be fixed once i solve the conflicts.

Reply:
Ok

Reply:
Done